dsl_survey/atlas_stack_language/predicates.yaml

```diff
@@ -0,0 +1,48 @@
+grammar:
+
+  - term: :eq
```

Comment:
Is the expectation that this would eventually get filled in with the complete set of operators?

Reply:
Yes, just starting with some examples to see if the format works for everyone.